server/db/migrations/1632132998306-RenameImageToImageURL.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
Not sure, but since we are introducing "image_url" for first time here we would not need a  migration  file?

Reply:
Nope, we're not using explicit migration files now, since typeorm can set up the db for us without them.  I'll remove this file while I sort out the conflicts.